GVFS/GVFS.Hooks/HooksPlatform/GVFSHooksPlatform.Mac.cs

```diff
@@ -1,42 +1,37 @@
-using GVFS.Platform.Mac;
+using GVFS.Platform.POSIX;

```

Comment:
Rename this file now? 

`GVFSHooksPlatform.Mac.cs` -> `GVFSHooksPlatform.POSIX.cs`

Reply:
Done.